README.md

```diff
@@ -221,7 +221,7 @@ Set up the test application to use your plugin. Open `.dev/package.json` and add
     "react": "^19.0.0",
     "react-dom": "^19.0.0",
     // Add this
-    "plugin-hello-world": "file:./"
+    "plugin-hello-world": "file:../"
```

Comment:
@lwjameson I believe this might have been the crucial bit that may have been missing. The Readme incorrectly pointed to the same dir, whereas it should of course point to the parent dir...